lib/zones.js

```diff
@@ -45,6 +45,21 @@ function read(zid, options) {
   });
 }
 
+function edit(zid, options) {
+  if (zid instanceof Zone) {
+    zid = zid.id;
+  }
+
+  var uri = join('zones', zid);
+
+  options.method = 'PATCH';
+  options.body = JSON.stringify(options.body);
```

Comment:
Look at how [`dns.js`](https://github.com/cloudflare/node-cloudflare/blob/4eabc388845c79dc0a56b98b4ddffaf6685e2597/lib/dns.js#L73) is done. The API expects an edited object to be passed in.

You'll probably need to implement `toJSON` here as well.